README.md

```diff
@@ -0,0 +1,133 @@
+## Memory Map
+Memory Map is a location-aware media archive that lets users upload photos, videos, and other files, automatically tagging them with time and GPS metadata, and visualizing them on an interactive world map.
+
+Users can browse the map, click markers, and explore media galleries tied to real-world locations — creating a digital memory atlas.
+
+## Features
+Upload media files (images, videos, documents)
+
+Automatic GPS location & timestamp tagging
+
+Interactive world map with clickable memory pins
+
+Gallery view for each map location
+
+GraphQL API backend
+
+Object storage powered by MinIO
+
+Rust backend with modern Nix dev-shell
+
+Fully browser-based frontend
+
+## Tech Stack
+Layer	Technology
+Frontend	React / Next.js
+Backend	Rust + GraphQL
+Storage	MinIO
+Database	PostgreSQL
+Dev Env	Nix + direnv
+Task Runner	Just
```

Comment:
1. Please format this as a table. Refer to the [GitHub Markdown formatting documentation](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables) for examples on how to do this.
2. The frontend doesn't use React/Next.js. We are using Leptos.